tests/upgrade/test_crossgrade.sh

```diff
@@ -104,6 +104,7 @@ ISTIO_ROOT=${GOPATH}/src/istio.io/istio
 TMP_DIR=/tmp/istio_upgrade_test
```

Comment:
Prob better to put all theses in $ARTIFACTS_DIR so these logs get uploaded.
e.g. http://gcsweb.k8s.io/gcs/istio-prow/pr-logs/pull/istio-releases_pipeline/238/release-upgrade-tests-1.0.2/256/

Reply:
these are just for temporary errors which are echoed to stdout in the test itself.